Draft.md

```diff
@@ -156,235 +156,171 @@ Eine Personengruppe, die besonders oft zum passiven Spielball von Forschungssoft
 
 #### Wissenschaftlicher Zweck und Beurteilung der Korrektheit
 
 * **Welches Problem versucht die Software zu lösen?** Welche Aufgaben werden mittels der Software bei der Erfassung, der Verarbeitung oder der Analyse von Daten durchgeführt? Wie sind diese Probleme und Aufgaben von ihrer Relevanz und von ihrer Häufigkeit in einem archäologischem Kontext einzuschätzen? z.B. "Schätzung von Bevölkerungsdichte aus Siedlungsgröße" oder "Statikmodellierung für Langhäuser"
 
-* **Gibt es Archäologie-relevante erfolgreiche Projekte/Anwendungen, die von dieser Software bereits unterstützt wurden?** Wie plausibel sind die erzielten Resultate? Wurden Ergebnisse der Projekte kritisiert, die insbesondere auf jene Ergebnisse beruhen, die mit der Software erzielt wurden?
+* **Wie löst die Software das gegebene Problem?** Wie ist die grundsätzliche Funktionsweise konzipiert? Was sind die wesentlichen Bestandteile in Frontend und Backend? z.B. "WebApp als Schnittstelle zu einer Datenbank" oder "Command line interface um ein Neuronales Netz für Nachhersage zu trainieren"
 
 * **Wie funktioniert der (wissenschaftliche) Algorithmus, der mit der Software implementiert wurde?** Welche wesentlichen Schritte durchlaufen Daten, um das Ausgangsproblem zu lösen? z.B. "Die Eingabedaten werden mittels eines Referenzdatensatzes bereinigt, mit hierarchischer Clusteranalyse klassifiziert und zuletzt als paarweise Distanzmatrix visualisiert"
 
-* **Wie löst die Software das gegebene Problem?** Wie ist die grundsätzliche Funktionsweise konzipiert? Was sind die wesentlichen Bestandteile in Frontend und Backend? z.B. "WebApp als Schnittstelle zu einer Datenbank" oder "Command line interface um ein Neuronales Netz für Nachhersage zu trainieren"
-
-* **Wie ist die Software publiziert?** Eignet sich die Software für wissenschaftliche Anwendungen, da sie in einer zitierbaren Publikation vorgestellt wurde? Sind einzelne Versionen klar als solche referenzierbar (etwa mittles DOI)? z.B. "Das Softwarepaket wurde in einer kurzen, zweiseitigen Publikation im Journal *Open Source Software* vorgestellt, diese Publikation schließt aber keine umfassenden Tests oder Vergleiche mit Alternativprodukten ein"
+* **Ist die Behauptung, eine bestimmte wissenschaftliche Fragestellung mit dem gewähltem Algorithmus beantworten zu können, korrekt?** Welche semantischen Schlüsse lassen sich aus rohen und verarbeiteten tatsächlich ableiten? Wie oben ausgeführt mag diese Frage in der Rezension nicht zu beantworten sein.
 
```

Comment:
"Welche semantischen Schlüsse lassen sich aus rohen und verarbeiteten tatsächlich ableiten? Wie oben ausgeführt mag diese Frage in der Rezension nicht zu beantworten sein." Ich finde das klingt sehr kompliziert (es fehlt auch ein Wort) und vielleicht muss hier auch rein, was weiter oben in der Einleitung zum Fragenkatalog steht? "Zum anderen ist zu beurteilen, ob Software eine korrekte Umsetzung der angestrebten Arbeit leistet. Letzteres, also die Korrektheit der Ergebnisse einer Software, ist mitunter schwer zu beurteilen, insbesondere dann, wenn es keine weiteren Implementierungen des Algorithmus in anderen Softwarepaketen gibt. Dieses Problem gliedert sich in mehrere Aspekte: Zunächst stellt sich die sehr technische Frage, ob die in der Dokumentation genannten Algorithmen fehlerfrei in Programmcode ausgedrückt wurden. Meist ist schon eine Prüfung dieses Aspekts nicht trivial und übersteigt den Rahmen einer Rezension. Darüber hinaus versprechen wissenschaftliche Softwarewerkzeuge implizit oder explizit die Beantwortung wissenschaftlicher Fragestellungen zu ermöglichen oder zumindest zu vereinfachen. Eine GIS-Applikation zur Analyse von Punktmustern kann etwa für den Zweck konzipiert sein, menschliches Siedlungsverhalten nachherzusagen. Die Frage, ob der Algorithmus dazu semantisch überhaupt in der Lage ist, kann den Umfang einer Rezension bei weitem übersteigen. Es bleibt aber bei der Rezensentin, eine Einschätzung der Plausibilität vorzunehmen und auch das Problem der Einschätzbarkeit selbst offenzulegen, um so die Leser dafür zu sensibilisieren."

Reply:
Naja - wiederholen sollten wir uns nicht. Das steht ja oben schon. Entweder wie ziehen das von oben herunter, oder wir belassen es bei dieser Notiz. Herunterziehen würde mir besser gefallen.

Reply:
Ja genau, ich fände runterziehen auch sehr gut. Machst Du es?

Reply:
Erledigt.